<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsInjectorDialog</class>
 <widget class="QDialog" name="SettingsInjectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Injector Settings</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>10</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="SettingsBox">
    <item row="0" column="1">
     <widget class="QLabel" name="inj1_label">
      <property name="text">
       <string>Injector 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="inj2_label">
      <property name="text">
       <string>Injector 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="inj1_BubbleVolume">
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="inj3_label">
      <property name="text">
       <string>Injector 3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>Injector Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="inj1_name">
      <property name="text">
       <string>Forskolin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="inj3EjectVolume"/>
    </item>
    <item row="4" column="2">
     <widget class="QDoubleSpinBox" name="inj2EjectVolume"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="inj2_name">
      <property name="text">
       <string>IBMX</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="inj3_name">
      <property name="text">
       <string>IsoProterenol</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="speed_label">
      <property name="text">
       <string>Injector Speed</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="inj1_Speed">
      <item>
       <property name="text">
        <string>Fastest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slowest</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="inj2_Speed">
      <item>
       <property name="text">
        <string>Fastest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slowest</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="inj3_Speed">
      <item>
       <property name="text">
        <string>Fastest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Slowest</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>PWM Up (µs)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="bubbleV_label">
      <property name="text">
       <string>Bubble Volume (µL)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="inj3_BubbleVolume">
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="inj1EjectVolume"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="ejectV_label">
      <property name="text">
       <string>Eject Mix Volume (µL)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QDoubleSpinBox" name="inj2_BubbleVolume">
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>PWM Down (µs)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="inj1PwmDown">
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1800</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="inj1PwmUp">
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="inj2PwmUp">
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="inj2PwmDown">
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1800</number>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="inj3PwmUp">
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QSpinBox" name="inj3PwmDown">
      <property name="maximum">
       <number>2000</number>
      </property>
      <property name="value">
       <number>1800</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
